### DISCUSSION TITLE
Craft 4 update

### DISCUSSION THREAD

**Discussion by olivierbon** [verbb/concierge#20] [General]
I had a few comments about this, and I thought I'd share my thinking publicly.

Now that Craft has **Settings → Users → Settings → Deactivate users by default** and the ability to create custom (filtered) Element index, I feel like Concierge is getting closed to being obsolete.

The only thing the plugin _really_ does is send notifications which could be done via a module.

I can make the plugin replicate Craft's native functionality in terms of activation/de-activation, but this doesn't seem like a good idea.

So, my question. Is it worth having a $15 plugin whose only purpose is sending notifications? Should Concierge be discontinued/open sourced so you can all do what you need with it?

I am looking forward to your comments!

**Comment by angieherrera** [comment]
Thanks for explaining this! I'll have to dive into Craft's settings to get a feel for it. This will take a couple days though for me to evaluate properly. That said, just from a gut feeling, I tend to agree with replicating functionality probably isn't a good idea.

**Comment by mattias78** [comment]
Thanks for the update Oli, personally I use the plugin just for the notification functionality (for site admins), I know you mentioned a module - which I did originally attempt before I installed this plugin - but I found it really difficult and documentation was sparse on how to write such a module (especially the email component). I feel that really if Craft is going to have the option to deactivate users by default shouldn't it have an option to also be notifying admin when users register? Maybe there's a reason why this isn't offered?

**Comment by outline4** [comment]
I just need it to send a confirmation email to the admins. 
It's a big site with loads of people involved and they tend to forget to check new registrations. 

Is there another way to send an email to the admins upon registration?  

I know I won't be able to create my own plugin.  

It's sad you want to discontinue it. I would still pay the 15$, but I would also prefer something native...

**Comment by mattias78** [comment]
Hey @outline4 a possible work around for you is to use Solspaces Freeform Pro plugin, you can create a front end user registration form and then send whatever email notifications you like https://docs.solspace.com/craft/freeform/v4/guides/user-registration-forms/

**Reply by outline4** [reply]
Thanks for the response. I was just about to try a combination of ryssbowh/craft-triggers and ryssbowh/craft-emails, but since I have freeform and the above is rather buggy, I will try with freeform... Thanks a lot!

**Reply by outline4** [reply]
and it works like a charm!

**Comment by engram-design** [comment]
Just joining in the conversation here, as we've used Concierge for a number of sites, where it's been great to allow us to notify clients when users register.

I think making this a free plugin is a good option, to just handle notifications. Related to that - we've added this to our [Craft 4-5 migration project](https://verbb.io/craft-4), as people are keen to see this plugin on Craft 5.

If you're no longer developing this plugin, we'd be happy to continue development at Verbb (including updating to Craft 5) - but just let us know if you've got plans to continue things and update to Craft 4 and 5! We'd plan to make it free.
